src/app/dataset-block/metadata-block/components/event-details/supported.events.ts

```diff
@@ -0,0 +1,11 @@
+export enum SupportedEvents {
+    AddData,
```

Comment:
Please leave only AddData for now.
We will be adding 1 by 1, once support is implemented.

Reply:
Fixed